experiments.json

```diff
@@ -60,5 +60,19 @@
       "min": "75",
       "max": "100"
     }
-  }
+  },
+  "promote-add-to-homescreen": {
+    "match": {
+      "appId": "^org.mozilla.fennec$"
+    },
+    "buckets": {
+      "min": 50,
+      "max": 100
+    },
+    "values": {
```

Comment:
I have not redeployed the server since that update, so Margaret is
absolutely right that values will not work yet - I will do that later today!

On Mon, Apr 11, 2016 at 7:39 AM, Margaret Leibovic <notifications@github.com
> wrote:

> In experiments.json
> <https://github.com/mozilla-services/switchboard-experiments/pull/28#discussion_r59215437>
> :
>
> > @@ -60,5 +60,19 @@
> >        "min": "75",
> >        "max": "100"
> >      }
> > -  }
> > +  },
> > +  "promote-add-to-homescreen": {
> > +    "match": {
> > +      "appId": "^org.mozilla.fennec$"
> > +    },
> > +    "buckets": {
> > +      "min": 50,
> > +      "max": 100
> > +    },
> > +    "values": {
>
> We have to make sure the server is update to actually support values :)
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla-services/switchboard-experiments/pull/28/files/510caea60e35e6696d85f3435b23717dcdbd1220#r59215437>
>